index.html

```diff
@@ -391,7 +393,8 @@ <h3 class="bannerbox__title">I can offer a ride</h3>
 
                         <div class="form-group checkbox">
                             <label for="officialDriver">
-                                <input type="checkbox" id="officialDriver" name="DrivingOnBehalfOfOrganization" /> I am driving in an official capacity on behalf of a registered organization
+                              <input type="checkbox" class="hidden" name="V" value="B" style="display:none;">
+                                <input type="checkbox" id="officialDriver" name="DrivingOnBehalfOfOrganization" style="display:none;" class="hidden" /> <!--I am driving in an official capacity on behalf of a registered organization -->
```

Comment:
okay so here's the plan: 

- Can you please delete the `officialDriver` AND `quesDriverOrgName` form group rows? i.e. the following code:
```
<div class="form-group checkbox">
    <label for="officialDriver">
        <input type="checkbox" id="officialDriver" name="DrivingOnBehalfOfOrganization" /> I am driving in an official capacity on behalf of a registered organization
    </label>
</div>
<div class="form-group" id="quesDriverOrgName" style="display:none">
    <label for="officialDriverOrgName">Organization name</label>
    <input type="text" class="form-input" id="officialDriverOrgName" name="DrivingOBOOrganizationName" data-error="The organisation name is required if the above box is checked.">
    <div class="help-block with-errors"></div>
</div>
```
- Please also delete the JS listener which toggles the org name row on/off:
```
// Toggle Official Driver input when checkbox is ticked:
$('#officialDriver').on('change', function() {
    var isOfficialDriver = $(this).is(':checked');
    $('#quesDriverOrgName').toggle(isOfficialDriver)
        .find('#officialDriverOrgName').prop('required', isOfficialDriver);
});
```